src/itest/java/uk/gov/companieshouse/exemptions/delta/util/ExemptionsRequestMatcher.java

```diff
@@ -0,0 +1,82 @@
+package uk.gov.companieshouse.exemptions.delta.util;
+
+import com.fasterxml.jackson.core.JsonProcessingException;
+import com.fasterxml.jackson.databind.JsonNode;
+import com.fasterxml.jackson.databind.ObjectMapper;
+import com.github.tomakehurst.wiremock.http.Request;
+import com.github.tomakehurst.wiremock.http.RequestMethod;
+import com.github.tomakehurst.wiremock.matching.MatchResult;
+import com.github.tomakehurst.wiremock.matching.ValueMatcher;
+import uk.gov.companieshouse.logging.Logger;
+
+public class ExemptionsRequestMatcher implements ValueMatcher<Request> {
+
+    private String expectedOutput;
+    private Logger logger;
+
+    public ExemptionsRequestMatcher(Logger logger, String output) {
+        this.expectedOutput = output;
+        this.logger = logger;
+    }
+
+    @Override
+    public MatchResult match(Request value) {
+
+        return MatchResult.aggregate(matchUrl(value.getUrl()), matchMethod(value.getMethod()),
+                matchBody(value.getBodyAsString()));
+    }
+
+    private MatchResult matchUrl(String actualUrl) {
+        String expectedUrl = "/company-exemptions/00006400/internal";
+
+        MatchResult urlResult = MatchResult.of(expectedUrl.equals(actualUrl));
+
+        if (! urlResult.isExactMatch()) {
+            logger.error("url does not match expected: <" + expectedUrl + "> actual: <" + actualUrl + ">");
```

Comment:
String concatenation in log messages is potentially expensive. Can these log lines be changed to this?
```
logger.error("url does not match expected: <%s> actual: <%s>", expectedUrl, actualUrl);
```

Reply:
I don't think we need this class anymore as it is not used anywhere for anything. Will delete.